src/containers/Notifications/notifications.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { fetchNotifications } from "../../redux/actions/notificationActions";
+
+export class Notifications extends Component {
+  componentDidMount() {
+    const username = localStorage.getItem("username");
+    const url =
+      "ws://ah-premier-staging.herokuapp.com/api/notifications/" +
+      username +
+      "/";
+
+    this.props.fetchNotifications(url);
+  }
+
+  notificationList = notifications => {
+    return notifications.map((notification, i) => {
+      return notification ? (
```

Comment:
Why do we have this check here? 
If we are mapping through the list of notifications, I don't think it's necessary to check if a single notification exists.